config/filesystems.php

```diff
@@ -2,9 +2,9 @@
 
 return [
     'content' => [
-        'driver'             => 'content-file-storage',
-        'base_url'           => env('CONTENT_BASE_URL'),
-        'base_rest_api_url'  => env('CONTENT_BASE_REST_API_URL'),
-        'default_repository' => env('CONTENT_DEFAULT_REPOSITORY'),
+        'driver' => 'content-file-storage',
+        'pathPrefix' => env('FILESYSTEM_PATH_PREFIX'),
+        'restAPIUrl' => env('FILESYSTEM_REST_API_URL'),
+        'defaultRepository' => env('FILESYSTEM_DEFAULT_REPOSITORY'),
```

Comment:
Nisam pro toga da se env varijable ovako generaliziraju. I za aws imaš prefixano `AWS_`, mislim da je prije bilo bolje, da se kroz env odmah zna da idemo na content servis, a ne da je to nešto generalno za filesystem.

Također, config entry nije camel case, nego snake kako je i prije bio.